backend/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:alpine as builder
+RUN mkdir /build
+COPY . /build
+WORKDIR /build
+RUN go build -o main .
```

Comment:
Please add the command which runs the whole Golang test suite for your components.

Reply:
Why should we run all test suites upon docker build?

I consider running it during CI rather than when building docker image

Reply:
Yes, you are right about running Go tests on the CI environment. This is just for security so that you don't forget to manually test code changes before using the Docker image on the local machine. It would be a good idea to enable automatic testing already at the image build stage.

Reply:
Ok, we will add it